build_tools/waf_dynamo.py

```diff
@@ -25,12 +25,22 @@
     print ("You must define DYNAMO_HOME. Have you run './script/build.py shell' ?", file=sys.stderr)
     sys.exit(1)
 
+def import_lib(module_name, path):
+    import importlib
+    # Normally a finder would get you the loader and spec.
+    loader = importlib.machinery.SourceFileLoader(module_name, path)
+    spec = importlib.machinery.ModuleSpec(module_name, loader, origin=path)
+    # Basically what import does when there is no loader.create_module().
+    module = importlib.util.module_from_spec(spec)
+    # Now is the time to put the module in sys.modules if you want.
+    # How import initializes the module.
+    loader.exec_module(module)
+
```

Comment:
I want to refactor this but I think it's better to do it on the private branches that actually use the code.